### PR TITLE
Various small fixes user roles

### PR DESCRIPTION
**Pull request by WillTrem**
### This PR addresses various small issues across the app, mostly related to users :
- Removed Schema Selector from CronJobTabs (uses the main one from the settings page)
- Fix NavBar schema selector only showing the settings-related schemas (instead of all schema_access schemas(
- Fix permissions granted to web_anon by mistake
- Removed role column from user management table

### CODE REVIEW

File: UInnovateApp/src/components/Schema/MenuSchemaSelector.tsx [hunk 1]
```diff
@@ -11,18 +11,15 @@ import { useNavigate } from 'react-router-dom';
...
 		...new Set(vmd.getApplicationSchemas()
 			.map((schema) => schema.schema_name)
 			.filter((schema_name) => {
 				// Ensures that on LOGIN_BYPASS without being logged in, all the schemas show
 				if ((LOGIN_BYPASS && user === null) // Include if LOGIN_BYPASS enabled with no user logged in
-					|| (schema_access.includes(schema_name)) // Schema must be in schema_access list
-					&& (dbRole === Role.ADMIN // AND User must be an admin
-						||(schemaRoles[schema_name] === Role.CONFIG // OR User must have role configurator for schema in schema roles
-							|| (!schemaRoles[schema_name] && defaultRole === Role.CONFIG) // OR User doesn't have any role set for schema and its default role is configurator
-						))) {
```
Comment (witnick) [inline UInnovateApp/src/components/Schema/MenuSchemaSelector.tsx:25]: I'm wondering if the same needs to be applied to Settings.tsx? if not please ignore this comment.

**Review by witnick — CHANGES_REQUESTED** [review]
see my previous comments for the things to resolve but otherwise logic change and UI update are working

File: UInnovateApp/src/components/Schema/MenuSchemaSelector.tsx (hunk 1 above)
Comment (WillTrem) [reply UInnovateApp/src/components/Schema/MenuSchemaSelector.tsx:25]: The logic is correct on the other schema selector in the settings page! The one in the NavBar depends solely on the schema_access list of the user, whereas the settings one depends also on the schema and default roles of the user, hence the added logic :)

File: database/meta.sql [hunk 2]
```diff
@@ -472,9 +472,9 @@ GRANT ALL ON meta.scripts TO configurator;
 GRANT ALL ON meta.i18n_languages TO configurator;
 GRANT ALL ON meta.i18n_keys TO configurator;
 GRANT ALL ON meta.i18n_values TO configurator;
-GRANT ALL ON meta.view_type TO web_anon;
-GRANT ALL ON meta.additional_view_settings TO web_anon;
-GRANT ALL ON meta.custom_view_templates TO web_anon;
+GRANT ALL ON meta.view_type TO configurator;
+GRANT ALL ON meta.additional_view_settings TO configurator;
+GRANT ALL ON meta.custom_view_templates TO configurator;
```
[the remark this answers is not in the document]
Comment (WillTrem) [reply database/meta.sql:477]: Agreed, will change!

[the remark this answers is not in the document]
Comment (WillTrem) [reply database/meta.sql:480]: Fixed @witnick !

**Review by witnick — DISMISSED** [review]
Approved ✅

**Review by witnick — APPROVED** [review]

**Pull request merged**